app/remoteCam/video.py

```diff
@@ -14,10 +14,10 @@
 
 if __name__ == "__main__":
 #    irled = led.sensor(3)  # Instantiate led class and assign the pin the BCM3
-    of = "/home/pi/IntermediateVideos/" # output folder
-    ff = "/home/pi/Videos/" # final folder
-    of1 = of + "1stPASS.mp4"
-    rectime = "10"  # record time of 10s
+    working_folder = "/home/pi/IntermediateVideos/" # output folder
+    final_folder = "/home/pi/Videos/" # final folder
+    output_file1 = working_folder + "1stPASS.mp4"
+    rectime = "10"  # record time working_folder 10s
```

Comment:
I think you did a find-and-replace here and this should still read "record time of 10s"?!

Reply:
Yep will fix